packages/react-provider/src/components/FluentProvider/useFluentProvider.ts

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { makeMergeProps, resolveShorthandProps, useMergedRefs } from '@fluentui/react-utilities';
+import { FluentProviderProps, FluentProviderState } from './FluentProvider.types';
+import { useFluent } from '@fluentui/react-shared-contexts';
+
+export const fluentProviderShorthandProps: (keyof FluentProviderProps)[] = [];
+
+const mergeProps = makeMergeProps<FluentProviderState>({ deepMerge: fluentProviderShorthandProps });
+
+/**
+ * Create the state required to render FluentProvider.
+ *
+ * The returned state can be modified with hooks such as useFluentProviderStyles,
+ * before being passed to renderFluentProvider.
+ *
+ * @param props - props from this instance of FluentProvider
+ * @param ref - reference to root HTMLElement of FluentProvider
+ * @param defaultProps - (optional) default prop values provided by the implementing type
+ *
+ * {@docCategory FluentProvider }
```

Comment:
This needs to be above the `@param` tags to work properly. And actually the docCategory tags aren't being used right now in convergence (they're specific to some doc-related features of the v8 website), so maybe we should just remove them all-up.

Reply:
I'm ok on removing it, but right now if I do so the conformance test simply fails,
![image](https://user-images.githubusercontent.com/5483269/112817452-bdd60480-9082-11eb-8a46-b067e0238bfb.png)

Reply:
Well, for the moment I'll disable `has-docblock` tests than

Reply:
I didn't mean you need to remove the whole doc comment, just move the `{@docCategory FluentProvider}` tag up above the `@param` parts (just below the main description).